test/elixir_sense/docs_test.exs

```diff
@@ -84,6 +84,55 @@ defmodule ElixirSense.DocsTest do
              """
     end
 
+    test "retrieve erlang function documentation edoc" do
+      buffer = """
+      defmodule MyModule do
+        def func(list) do
+          :edoc.file(list, "")
+        end
+      end
+      """
+
+      %{
+        subject: subject,
+        actual_subject: actual_subject,
+        docs: %{docs: docs}
+      } = ElixirSense.docs(buffer, 3, 12)
+
+      assert subject == ":edoc.file"
+      assert actual_subject == ":edoc.file"
+
+      assert docs =~ """
+             > :edoc.file(term, term)
+
+             Reads a source code file and outputs formatted documentation to
+             a corresponding file.
+
+             Options:
+               {dir, filename()}
+
+                   Specifies the output directory for the created file. (By
+                   default, the output is written to the directory of the source
+                   file.)
+
+               {source_suffix, string()}
+
+                   Specifies the expected suffix of the input file. The default
+                   value is ".erl".
+
+               {file_suffix, string()}
+
+                   Specifies the suffix for the created file. The default value is
+                   ".html".
+
+             See get_doc/2 and layout/2 for further
+             options.
+
+             For running EDoc from a Makefile or similar, see
+             edoc_run:file/1.
+             """
```

Comment:
Is there a way to not depend on the real docs? Is it possible to have samples files with docs locally for tests just like we have for Elixir in `test/support`? Otherwise, we'll face the old problem of lots of failing tests whenever they change the docs.

Reply:
I've created a separate issue https://github.com/elixir-lsp/elixir_sense/issues/91